src/driver/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+option(BUILD_ACQUIRE_DRIVER_ZARR "Build the Acquire Zarr driver" ON)
```

Comment:
Nit: It makes more sense to set this option in the root CMakeLists file and conditionally add the driver's subdirectory, rather than wrapping this entire configuration file in an if statement.

Reply:
Do you mean the top level or the one in src/?